main.0bab88e4d3fcc53983b1.js

```diff
@@ -127379,9 +127379,9 @@ var _generic = __webpack_require__(12);
 var token = (0, _generic.getToken)();
 
 var CONFIG = {
-  BASE_URL: window.location.href.split('/#')[0] || 'http://hr.excellencetechnologies.in/',
+  BASE_URL: window.location.href.split('/#')[0] || 'localhost/hr/',
```

Comment:
these should be as it was earlier.
do changes here nly for your use when development is going on